### DISCUSSION TITLE
UDF

### DISCUSSION THREAD

**Discussion by zxding** [apache/doris#23611] [RFC]
Can I get loginsession and current database info which call this UDF when I define the user defined function

**Comment by zxding** [comment]
自定义Udf 解密函数，密钥需要根据用户以及当前库的信息从三方密钥库授权获取密钥信息完成解密，无法通过定义udf参数形式传递密钥，请问udf函数是否提供一个上下文参数，在初始化的时候可以获取相关登录用户信息

**Reply by lsy3993** [reply]
Not supported yet, Maybe we can add it
